### PR TITLE
Matching Julius and Augustus Translation

### PR DESCRIPTION
**Pull request by webfischi**
Also added the 2 new Hotkey Strings

### CODE REVIEW

File: src/translation/german.c [hunk 1]
```diff
@@ -4,10 +4,10 @@
 static translation_string all_strings[] = {
     {TR_NO_PATCH_TITLE, "Patch 1.0.1.0 ist nicht installiert"},
     {TR_NO_PATCH_MESSAGE,
-        "Deine Caesar 3 Installation hat den Patch 1.0.1.0 nicht installiert. "
+        "DDeine Caesar 3 Installation wurde noch nicht auf Version 1.0.1.0 aktualisiert. "
```
Comment (crudelios) [inline src/translation/german.c:7]: Double D
```suggestion
        "Deine Caesar 3 Installation wurde noch nicht auf Version 1.0.1.0 aktualisiert. "
```

File: src/translation/german.c [hunk 2]
```diff
@@ -32,63 +32,64 @@ static translation_string all_strings[] = {
...
+    {TR_HOTKEY_ALTERNATIVE_LABEL, "Alternativ"},
     {TR_HOTKEY_HEADER_ARROWS, "Pfeiltasten"},
     {TR_HOTKEY_HEADER_GLOBAL, "Globale Tastenkombinationen"},
     {TR_HOTKEY_HEADER_CITY, "Stadt"},
     {TR_HOTKEY_HEADER_ADVISORS, "Berater"},
     {TR_HOTKEY_HEADER_OVERLAYS, "Spezialkarten"},
     {TR_HOTKEY_HEADER_BOOKMARKS, "Lesezeichen für Stadtkarten"},
     {TR_HOTKEY_HEADER_EDITOR, "Editor"},
-    {TR_HOTKEY_HEADER_BUILD, "Bauen"},
+    {TR_HOTKEY_HEADER_BUILD, "Tastenkombinationen zum Bauen"},
```
Comment (crudelios) [inline src/translation/german.c:60]: Did you check if this string fits?

**Review by crudelios — CHANGES_REQUESTED** [review]
I had a look, just some slight issues to fix 👍

File: src/translation/german.c (hunk 2 above)
Comment (webfischi) [reply src/translation/german.c:60]: I didn't check this Version, but the one I did in Augustus previously had still space left.

**Pull request merged**